document_merge_service/api/views.py

```diff
@@ -98,3 +80,28 @@ def retrieve(self, request, **kwargs):
         response["Content-Length"] = template.template.size
         response.write(template.template.read())
         return response
+
+
+class ConvertView(APIView):
+    def post(self, request, **kwargs):
+        serializer = serializers.ConvertSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        file = serializer.data["file"]
+        target_format = serializer.data["target_format"]
+
+        content_type, foo = mimetypes.guess_type(file.name)
+
+        if (
+            content_type
+            != "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
+        ):
```

Comment:
This could possibly be extended to support more office formats (specifically, ODT as LibreOffice's native format)